e2e/regression_test/cgm.data_loss.spec.js

```diff
@@ -0,0 +1,32 @@
+const { setup, loopSettings, screen, SettingDefault, CGMModel } = require('../../src/index');
+
+describe.skip('Closed loop is stopped when we have cgm data loss', () => {
+    let config = {
+        scenario: 'flat_cgm_trace',
+        settings: SettingDefault,
+    };
+    beforeAll(async () => {
+        await setup.LaunchLoop();
```

Comment:
[nit] Is there any way to push the `beforeAll` and `afterAll` globally? I imagine we need to run these two for *any* test. (Or is that not the case?)